src/app/components/dashboard/dashboard.component.ts

```diff
@@ -454,6 +454,32 @@ export class DashboardComponent implements OnInit {
 
 		this.chartError = false
 
+		// force timestamp to be at 00:00AM for the day to keep columns centered on ticks
+		for (let i = 0; i < this.chartData.consensusChartData.length; i++) {
```

Comment:
Minor feedback: since we are using the code for both arrays, make it a function. Otherwise lgtm